app/early-access/page.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/navigation'
+import PrelaunchSignUpForm from '../components/main-content/PrelaunchSignUpForm';
+import SubmissionStatus from '../components/main-content/SubmissionStatusPopup';
+
+type FlowState = 'viewingForm' |'processing' | 'confirmed';
+/**
+ * Handles the flow from viewing the signup form, processing the form submission,
+ * to confirming submission success, followed by redirection to the homepage.
+ */
+export default function PreLaunchSignupFlowContainer() {
+  const [flowState, setFlowState] = useState<FlowState>('viewingForm');
+  const router = useRouter()
+
+  const handleFormSubmit = () : void => {
+    setFlowState('processing');
+  // TODO: Expand with form submission logic & refactor setTimeOut placeholder
+     setTimeout(() => setFlowState('confirmed'), 1000); // Placeholder for actual submission logic.
+    scheduleRedirectHome();
```

Comment:
I think invoking `scheduleRedirectHome` here is unnecessary since the useEffect will do so after `flowState` is set to 'confirmed'

Reply:
True. I just realized I have duplicated the set timeout.